### PR TITLE
Jkmarx/node group api reformat

### LINKED ISSUE #1244: Node Group API - Current Selection First
The default current selection node group should be the first item in array.
The api should sort the current selection before sending response.

Ref #1038

### PR DESCRIPTION
**Pull request by jkmarx**
Resolves #1244

### CODE REVIEW

File: refinery/core/utils.py [hunk 1]
```diff
@@ -827,3 +827,24 @@ def create_delete_response_message(del_response):
...
+def move_obj_to_front(obj_arr, prop_key, prop_value):
+    """
+        Helper method move the first obj matching to the front of the arr
+        based on key and value
+        :param objects_to_delete: iterable of Objects selected in admin UI,
+        or a single object instance if `delete_model:admin_ui_deletion` is
+        called with `single_model` having a truthy value
+        :param obj_arr: An array of objects
+        :param prop_key: Key to match
+        :param prop_key: Value to match
```
Comment (scottx611x) [inline refinery/core/utils.py]: checkout typo `:param prop_key:` x 2

File: refinery/core/utils.py [hunk 2]
```diff
@@ -827,3 +827,24 @@ def create_delete_response_message(del_response):
             # Fix for multiple messages displaying
             messages.set_level(request, messages.ERROR)
             create_delete_response_message(delete_response)
+
+
+def move_obj_to_front(obj_arr, prop_key, prop_value):
```
Comment (scottx611x) [inline refinery/core/utils.py]: Can we rename `prop_key` and `prop_value` to something like `key_to_match`  & `value_to_match`??

Comment (jkmarx) [inline refinery/core/utils.py]: @scottx611x How about match_key and match_value?

Comment (scottx611x) [inline refinery/core/utils.py]: @jkmarx sounds good to me 👍

**Pull request merged**